src/Microsoft.ML.LightGBM/LightGbmTrainerBase.cs

```diff
@@ -163,7 +164,7 @@ private protected virtual void CheckDataValid(IChannel ch, RoleMappedData data)
             ch.CheckParam(data.Schema.Label.HasValue, nameof(data), "Need a label column");
         }
 
-        protected virtual void GetDefaultParameters(IChannel ch, int numRow, bool hasCategarical, int totalCats, bool hiddenMsg=false)
+        protected virtual void GetDefaultParameters(IChannel ch, int numRow, bool hasCategarical, int totalCats, bool hiddenMsg = false)
```

Comment:
>hiddenMsg [](start = 118, length = 9)

for the sake of not having short names -hiddenMessage?

Reply:
I'll leave that stuff to whoever would scrub lightgbm learners.